bin/clojupyter.ps1

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
For the record I'm unable to review PS so I'm merging this on faith that you've tested :)

Reply:
I've tested the powershell scripts in a Windows VM. Windows is not my main OS though, so the development usually followed the bash scripts.

One potential issue I can think about are the symlinks we create. In bash we make a symlink to the plugin jar to enable it and we do the same thing in PS. However, I don't think regular users are allowed to create symlinks in Windows by default. So it might be better to just copy the jar over instead of using symlinks.